modules/shared/components/atoms/avatar/utils.ts

```diff
@@ -0,0 +1,12 @@
+export const determineAvatarSize = (componentSize: string): number => {
```

Comment:
- we need to add this function inside `/shared/logic/[folderName]` so that we are consistent throughout the whole project
- Adding test cases for the function will be very helpful

Reply:
We have talked about that naming convention and I think we settled in naming it `Avatar.tsx` instead of `index.tsx`.

Reply:
There are test cases for the function in `Avatar.test.tsx` already.